### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by PemaYT**
sign up form - not 100% sure about mobile compatibility, please check :)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -18,26 +17,81 @@ body {
...
+    color: white;
+}
 
+.terms {
+    color: gray;
+    font-size: 10px;
+}
+
+.bold {
+    color: red;
```
Comment (Zyanvel) [inline styles.css]: Nit: I would advice to use a color picker and select the color from the design :) just for practice

**Review by Zyanvel — CHANGES_REQUESTED** [review]
Nice job! Left one nit comment and it's just matter to review the width for the mobile screen.

**Review by Zyanvel — APPROVED** [review]

**Comment by PemaYT** [comment]
one more ticket left hehe

**Comment by soleyingunn** [comment]
How can I open the code to review it?

**Comment by soleyingunn** [comment]
In my VS Code so I can inspect it I mean :) Sorry didn't mean to be unclear

**Comment by PemaYT** [comment]
sorry hahaha my bad

**Pull request merged**